client/src/plugins/user-journey-statistics/Util.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
Should we maybe add this as a general `util`? Or is something heavily related to the user journey statistics?

Reply:
We also use these for ET, maybe it makes sense to have it somewhere more central.